src/cascadia/TerminalApp/defaults.json

```
@@ -323,6 +323,23 @@
         { "command": "paste", "keys": "ctrl+shift+v" },
         { "command": "paste", "keys": "shift+insert" },
 
+        // Selection Modification (by cell)
+        { "command": { "action": "moveSelectionPoint", "direction": "down",  "expansionMode": "cell" },  "keys": ["shift+down"] },

```

nit: these could be 
```suggestion
        { "command": { "action": "moveSelectionPoint", "direction": "down",  "expansionMode": "cell" },  "keys": "shift+down" },
```

style instead